integration-tests/src/main/java/io/druid/testing/ConfigFileConfigProvider.java

```diff
@@ -0,0 +1,116 @@
+/*
+ * Druid - a distributed column store.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.druid.testing;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.io.FileNotFoundException;
+import java.io.File;
+import java.io.IOException;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import java.util.Map;
+
+public class ConfigFileConfigProvider implements IntegrationTestingConfigProvider
+{
+  private String configFile;
+
+  private String routerHost = "";
+  private String brokerHost = "";
+  private String coordinatorHost = "";
+  private String indexerHost = "";
+  private String middleManagerHost = "";
+  private String zookeeperHosts = "";
+  private Map<String, String> props = null;
+
+  @JsonCreator
+  ConfigFileConfigProvider(@JsonProperty("configFile") String configFile){
+    this.configFile = configFile;
+    loadProperties();
+  }
+
+  private void loadProperties()
+  {
+    ObjectMapper jsonMapper = new ObjectMapper();
+    try {
+      props = jsonMapper.readValue(
+          new File(configFile), new TypeReference<Map<String, String>>()
+      {
+      }
+      );
+    }
+    catch (FileNotFoundException ex) {
+      throw new RuntimeException(ex.getMessage());
```

Comment:
can you wrap exception object instead of just the message, stacktrace is useful at times.